invoke-ai/InvokeAI
# feat(ui): trigger phrases

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

## Description

This is based off #5849, rebased on #5846. I was going to just do this on the original PR branch but I screwed something up and created this new branch, where I fixed conflicts and made some changes. Mary's branch was actually fine, I just made a typo and though it was broken. Whoops

Add trigger phrases to UI:
- Edit them via MM tab
- Use them by adding a LoRA and then pressing `<` in the prompt box

Changes from @maryhipp 's PR:
- Derive valid embeddings and LoRAs within the component. No need to do this in a listener and lift the prompt trigger options into global state.
- Also fixed an issue w/ display of model "metadata"

## QA Instructions, Screenshots, Recordings

- Add a LoRA and you should be able to get its trigger word with `<` in the prompt box.
- You should be able to add and remove trigger words from the model manager tab.



## Merge Plan

This PR can be merged when approved
